source/documentation/virtual_machine_management_guide/topics/Configuring_virtual_NUMA.adoc

```diff
@@ -22,13 +22,16 @@ If you define *NUMA Pinning*, the default migration mode is *Allow manual migrat
 . Click *NUMA Pinning*.
 . In the *NUMA Topology* window, click and drag virtual NUMA nodes from the box on the right to host NUMA nodes on the left as required, and click btn:[OK].
 . Select *Strict*, *Preferred*, or *Interleave* from the *Tune Mode* drop-down list in each NUMA node. If the selected mode is *Preferred*, the *NUMA Node Count* must be set to `1`.
-. You can also set the NUMA pinning policy automatically by selecting an *Auto Pinning Policy* from the drop-down list:
+. You can also set the NUMA pinning policy automatically by selecting *Resize and Pin NUMA* from the CPU Pinning Polcy drop-down list under the *CPU Allocation* settings in the *Resource Allocation* tab:
 +
-* *None* - makes no changes to the virtual machine.
-//* *Pin* - uses the existing CPU topology that has been set, in order to configure CPU pinning and NUMA pinning.
-* *Resize and Pin* - maximizes the CPU topology and generates the CPU pinning and NUMA pinning configurations.
+* `None` - runs without any CPU pinning.
+* `Manual` - available only when the virtual machine is pinned to a Host.
+* `Resize and Pin NUMA` - resizes the virtual CPU and NUMA topology of the virtual machine according to the Host, and pins them to the Host resources.
+* `Dedicated` - the selected CPU pinning policy is available for cluster compatibility level 4.7 or later. If the virtual machine has NUMA enabled, all nodes must be unpinned.
+* `Isolate Threads` - the selected CPU pinning policy is available for cluster compatibility level 4.7 or later. If the virtual machine has NUMA enabled, all nodes must be unpinned.
```

Comment:
Worth adding what it actually does to the limitations? It also described in the UI when opening the drop-down list.
Manual - Runs manually specified virtual CPU on a specific physical CPU on a specific host.
Dedicated - Exclusively pin virtual CPUs to host physical CPUs.
Isolate Threads - Exclusively pin virtual CPUs to host physical CPUs. Each virtual CPU gets a physical core.

Without dropping the limitations above (host pinning, cluster level).

Unless it shouldn't be here and only the limitations are.